WordPress/src/main/res/values/strings.xml

```diff
@@ -1061,6 +1061,14 @@
     <string name="scan_history">Scan History</string>
     <string name="scan_history_menu_title">History</string>
 
+    <string name="scan_loading_title">Loading scan</string>
```

Comment:
🔍 Nitpick: We might want to consider adding "..." at the end of this string resource. Wdyt?

Reply:
Nitpick accepted, added '...' in cf7dfd6.